mock/integration-tests/20-lvm-plugin.tst

```diff
@@ -43,6 +43,27 @@ runcmd "$MOCKCMD --rebuild ${TESTDIR}/test-C-1.1-0.src.rpm --resultdir $TMPDIR"
 
 runcmd "$MOCKCMD --shell '/bin/true'" || die "mock shell failed"
 
+mkdir $TMPDIR/repodir || die "can't create repodir inside tmpdir"
+curl https://kojipkgs.fedoraproject.org//packages/python-copr-common/0.6/1.fc32/noarch/python3-copr-common-0.6-1.fc32.noarch.rpm \
+    --output $TMPDIR/repodir/python3-copr-common-0.6-1.fc32.noarch.rpm || die "failed to download old rpm package"
```

Comment:
The problem is that the `python3-copr-common` package is in the default fedora repository, and (especially in future) it is in higher version --- so this explicit variant likely won't be installed.

We need some trivial spec file, and build a custom RPM with artificial NVR here ... which will be later (before the image update) bumped once more.

Reply:
I created a `mock-test-bump-version` package, [v1 src.rpm](https://raw.githubusercontent.com/rpm-software-management/mock-test-data/master/mock-test-bump-version-1-0.src.rpm) and [v2 src.rpm](https://raw.githubusercontent.com/rpm-software-management/mock-test-data/master/mock-test-bump-version-2-0.src.rpm).

Feel free to use this in pattern:
```
wget those files
mock <version1> --target --resultdir <repo>
createrepo_c <repo>
generate custom config <custom.cfg> with that repo + added the package in the minimal buildroot
mock -r <custom.cfg> --init
mock <version2> --target --resultdir <repo>
createrepo_c <repo>
mock -r <custom.cfg> --rebuild something # check that the package is updated, and cache regenerated
```